lib/vsc/filesystem/gpfs.py

```diff
@@ -391,10 +405,10 @@ def list_quota(self, devices=None):
 
         for idx, (fs, qt, qid) in enumerate(zip(info['filesystemName'], info['quotaType'], info['id'])):
             details = dict([(k, info[k][idx]) for k in datakeys])
-            if qt == 'FILESET':
+            if qt == Typ2Param.FILESET.value:
```

Comment:
use `self.quota_types`

Reply:
Done in https://github.com/hpcugent/vsc-filesystems/pull/83/commits/4f764b97cd1967265065bf023d3c17e932dcce97